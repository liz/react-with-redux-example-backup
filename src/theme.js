// eslint-disable-next-line import/no-webpack-loader-syntax
import variables from 'sass-extract-loader?{"plugins" :["sass-extract-js"]}!./variables.scss';

export default variables;