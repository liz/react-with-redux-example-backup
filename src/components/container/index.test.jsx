import React from 'react';
import { mount } from 'enzyme';

import { Container } from './';
import theme from '../../theme';

describe('Container', () => {
    it('should match the snapshot', () => {
        const wrapper = mount(<Container />);

        expect(wrapper.html()).toMatchSnapshot();
    });

    it('renders Container with expected props', () => {
        const wrapper = mount(<Container />);

        expect(wrapper.find('Container')).toHaveLength(1);
        expect(wrapper.find('Container').props()).toEqual({
            maxWidth: theme.container,
            horizontalPadding: theme.gutter,
            verticalPadding: null
        });
    });
});