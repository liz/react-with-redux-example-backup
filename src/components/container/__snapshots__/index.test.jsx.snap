// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container should match the snapshot 1`] = `"<div class=\\"container__Shell-sc-1efg81n-0 gepMUD\\"></div>"`;
